pkgs/os-specific/linux/firmware/ipu6-camera-bins/default.nix

```diff
@@ -4,26 +4,19 @@
 , autoPatchelfHook
 , expat
 , zlib
-
-# Pick one of
-# - ipu6 (Tiger Lake)
-# - ipu6ep (Alder Lake)
-, ipuVersion ? "ipu6"
 }:
 
 stdenv.mkDerivation (finalAttrs: {
-  pname = "${ipuVersion}-camera-bin";
```

Comment:
IMO this should be renamed to `ipu6-camera-bins` (same as the upstream project) since it includes all the bins.

Reply:
I'd just rename it like it is called upstream and update the references TBH.